framer/Animation.coffee

```diff
@@ -294,8 +296,14 @@ class exports.Animation extends BaseClass
 		@_target[key] = Color.mix(@_stateA[key], @_stateB[key], value, false, @options.colorModel)
 
 	_updateGradientValue: (key, value) =>
+		if not @_stateB[key] and value is 1
+			@_target[key] = @_stateB[key]
```

Comment:
Are we resetting the target value here? because `@_stateB[key]` is empty, right?